Makefile

```diff
@@ -289,6 +289,11 @@ init-db-test-from-prod: db_prod_dump.trig init-db-test-empty ## init local datab
 	@curl -X POST -H "Content-Type: application/sparql-update" -d "DROP ALL" -u "admin:test" "http://localhost:3030/knora-test"
 	@curl -X POST -H "Content-Type: application/trig" -T "${CURRENT_DIR}/db_prod_dump.trig" -u "admin:test" "http://localhost:3030/knora-test"
 
+.PHONY: init-db-test-from-dev
+init-db-test-from-dev: ## init local database with data from staging. Use as `make init-db-test-from-dev PW=database-password`
+	@echo $@
+	${MAKE} init-db-from-env PW=${PW} ENV=db.dev.dasch.swiss
```

Comment:
I wonder, does it actually need the `PW=${PW}` or would that propagate through anyways?